hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/CreateTopLevelDagVisitor.java

```diff
@@ -252,12 +259,13 @@ public Vertex onMapIndexScan(IndexScanMapPhysicalRel rel) {
     @Override
     public Vertex onCalc(CalcPhysicalRel rel) {
         RexProgram program = rel.getProgram();
-        List<Expression<?>> projection = rel.projection(parameterMetadata);
+        dagBuildContext.setTable(null);
```

Comment:
nit: is it really necessary to nullify table in context?

Reply:
Not strictly necessary, but it's done for the sake of completeness. I was thinking setting the table and rel-node with one call, that might be even better and easier to understand the design.